bitnami/contour/README.md

```diff
@@ -84,21 +85,21 @@ $ helm uninstall my-release
 | `contour.image.tag`                               | Contour image tag                                                                                                                  | `1.18.0-debian-10-r5`                           |
 | `contour.image.pullPolicy`                        | Contour Image pull policy                                                                                                          | `IfNotPresent`                                  |
-| `contour.image.pullSecrets`                       | Contour Image pull secrets                                                                                                         | `[]`                                            |
-| `contour.hostAliases`                             | Add deployment host aliases                                                                                                        | `[]`                                            |
-| `contour.extraArgs`                               | Extra arguments passed to Contour container                                                                                        | `[]`                                            |
-| `contour.resources.limits`                        | Specify resource limits which the container is not allowed to succeed.                                                             | `{}`                                            |
-| `contour.resources.requests`                      | Specify resource requests which the container needs to spawn.                                                                      | `{}`                                            |
+| `contour.image.pullSecrets`                       | Contour Image pull secrets                                                                                                         | `undefined`                                     |
+| `contour.hostAliases`                             | Add deployment host aliases                                                                                                        | `undefined`                                     |
+| `contour.extraArgs`                               | Extra arguments passed to Contour container                                                                                        | `undefined`                                     |
+| `contour.resources.limits`                        | Specify resource limits which the container is not allowed to succeed.                                                             | `undefined`                                     |
+| `contour.resources.requests`                      | Specify resource requests which the container needs to spawn.                                                                      | `undefined`                                     |
```

Comment:
We should fix all these "undefined" references. Do you mind if I contribute with a commit to your PR with the required changes to fix it?

Reply:
Don't mind, thanks